tests/baselines/reference/user/puppeteer.log

```diff
@@ -47,6 +47,8 @@ lib/Page.js(112,33): error TS2345: Argument of type 'CDPSession' is not assignab
 lib/Page.js(189,15): error TS2503: Cannot find namespace 'Protocol'.
 lib/Page.js(284,15): error TS2503: Cannot find namespace 'Protocol'.
 lib/Page.js(369,20): error TS2503: Cannot find namespace 'Protocol'.
+lib/Page.js(432,7): error TS2740: Type '(...args: any[]) => Promise<any>' is missing the following properties from type 'Window': Blob, TextDecoder, TextEncoder, URL, and 231 more.
+lib/Page.js(442,9): error TS2349: Cannot invoke an expression whose type lacks a call signature. Type 'Window' has no compatible call signatures.
```

Comment:
numeric index signature means that `window[x] : Window` where `x: any`. And the arrow in question is not assignable to Window.